modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './index.scss';
+
+import {useState} from 'react';
+
+import NavigationBar from '../../../../../common/components/navigation-bar';
+import {getWebDavUrl} from '../../../../../common/utils/webdav';
+
+const PolicyDetail = () => {
+	const amountOfPoliceDetail = [0];
+	const navbarLabel = ['Vehicles', 'Drives'];
```

Comment:
Please, use an enum for 'Vehicle' and 'Drives'

Reply:
Done!